clang/include/clang/Basic/LangOptions.def

```diff
@@ -470,6 +470,9 @@ ENUM_LANGOPT(StrictFlexArraysLevel, StrictFlexArraysLevelKind, 2,
 
 COMPATIBLE_VALUE_LANGOPT(MaxTokens, 32, 0, "Max number of tokens per TU or 0")
 
+BENIGN_LANGOPT(PointerAuthFunctionTypeDiscrimination, 1, 0,
```

Comment:
Can we use `LANGOPT` just as we use for similar `PointerAuth*` options above? Also, moving this definition to similar ones above would probably make sense so they are grouped together visually.